ons_alpha/core/management/commands/scheduler.py

```diff
@@ -35,4 +35,5 @@ def add_management_command(self, command_name, trigger, **kwargs):
 
     def configure_scheduler(self):
         # "second=0" run the task every minute, on the minute (ie when the seconds = 0)
-        self.add_management_command("publish_scheduled", CronTrigger(second=0))
+        self.add_management_command("publish_bundles", CronTrigger(second=0))
+        self.add_management_command("publish_scheduled_without_bundles", CronTrigger(minute=5))
```

Comment:
Question: Why are you only checking scheduled pages once an hour?

Reply:
gah, I wanted it to run every 5 min 🤦🏼 
will re-read the docs

Reply:
Updated to `*/5` as per https://apscheduler.readthedocs.io/en/3.x/modules/triggers/cron.html#expression-types